internal/gui/gui.go

```diff
@@ -182,19 +182,30 @@ func initVpnClientBtn(conf *visorconfig.V1, httpClient *http.Client, logger *log
 	rpcC = rpcClient(conf, rpc_logger)
 
 	mVPNClient = systray.AddMenuItem("VPN", "VPN Client Submenu")
+	mVPNClient.Disable()
+	go mVPNClientEnable(rpcC, logger.PackageLogger("systray:vpn_button"))
 	// VPN Status
 	mVPNStatus = mVPNClient.AddSubMenuItem("Status: Disconnected", "VPN Client Status")
 	mVPNStatus.Disable()
 	go vpnStatusBtn(conf, rpcC)
 	// VPN Connect/Disconnect Button
 	mVPNButton = mVPNClient.AddSubMenuItem("Connect", "VPN Client Switch Button")
 	// VPN Public Servers List
-	mVPNServersList := mVPNClient.AddSubMenuItem("Servers", "VPN Client Servers")
-	mVPNServers := []*systray.MenuItem{}
-	for _, server := range getAvailPublicVPNServers(conf, httpClient, logger.PackageLogger("systray:servers")) {
-		mVPNServers = append(mVPNServers, mVPNServersList.AddSubMenuItemCheckbox(server, "", false))
+	go serversBtn(conf, rpcC, httpClient, logger.PackageLogger("systray:vpn-servers"))
+}
+
+func mVPNClientEnable(rpcC visor.API, logger *logging.Logger) {
+	logger.Warn("Initializing...")
+	for {
+		isReady, _ := rpcC.IsDMSGClientReady()
+		if isReady {
+			mVPNClient.Enable()
+			logger.Info("Initialized.")
+			return
+		}
+		logger.Warn("DSMG-Client not ready yet, will check in next 10 seconds.")
```

Comment:
`DSMG-Client` is misspelled. `DMSG-Client`

Reply:
Fixed.